app/request/context.go

```diff
@@ -38,9 +38,14 @@ func NewContext(ctx context.Context, requestId, ipAddress, path, userAgent, acce
 	}
 }
 
-func EmptyContext() *Context {
+func EmptyContext(logger mlog.LoggerIFace) *Context {
+	if logger == nil {
+		logger, _ = mlog.NewLogger() // NewLogger without arguments never errors
+	}
```

Comment:
Do we still want to do this? I'd be inclined to _always_ pass a logger.

Reply:
I added it more as a error prevention measure. I have absolutely no problem removing it,

Reply:
Let's remove it. I'd prefer the code to crash if nil is passed.